### ISSUE TITLE
Jellyfin doesn't report an error if disk access is failing or taking a very long time

### ISSUE THREAD

**Issue by kaysond** [jellyfin/jellyfin#5076]
**Describe the bug**
If media and metadata (specifically images) is accessed via an nfs mount which is not fully operational, jellyfin chokes on some of the reads and the endpoints hang, but it produces no error message in the jellyfin logs

**System (please complete the following information):**
 - OS: Ubuntu 20.04
 - Virtualization: Docker
 - Clients: Browser
 - Browser: Firefox 83
 - Jellyfin Version: 10.6.4
 - Playback: [Direct Play, Remux, Direct Stream, Transcode] 
 - Installed Plugins: Anime
 - Reverse Proxy: apache
 - Base URL: none
 - Networking: NAT
 - Storage: NFS
 
**To Reproduce**

1. Place your media and images on an nfs mount with `local_lock=none`
2. Disable the locking daemon on the nfs server `rpc.statd`
3. Run jellyfin and access the web ui, loading any image thats stored on the nfs mount (like thumbnails)

**Expected behavior**
Jellyfin reports that it's having trouble accessing the disk

**Additional context**
See #4603 

TL;DR: disk access was sporadically failing, causing jellyfin to hang. it would be nice if jellyfin's reads had some timeout, especially for reads from nfs mounts, after which it logs a disk access error and closes the http response. Not sure if this is easy or practical in .net, but it wouldve saved me 2 months of debug!

**Comment by Bond-009** [comment]
Closing as stale
